pyuvsim/simsetup.py

```diff
@@ -508,9 +244,9 @@ def create_mock_catalog(time, arrangement='zenith', array_location=None, Nsrcs=N
     stokes = np.zeros((4, 1, Nsrcs))
     stokes[0, :] = fluxes
     freqs = np.ones(Nsrcs) * freq
```

Comment:
This is an older syntax that we want to avoid. The old definition of SkyModel gave each source component a reference frequency, which is why the init accepted an array of frequencies of length Nsrcs. The new SkyModel class (since we added support for healpix maps with explicit frequency structure) is initialized with a `freq_array`, which represents the array of frequencies corresponding with the given fluxes for each source.

It would be clearer to rename this `freq_array` and make it an array of length 1 -- `freq_array = np.array([freq])` -- then pass that into the SkyModel init.

Reply:
Hmm, the shape of that array is currently supposed to be length `NComponents` in the master branch of pyradiosky, so I'll have to fix it there. What shape should it be?

Reply:
It should just be length `Nfreqs` (in this case, 1). Unless we decided otherwise, we were using one frequency array for all components.

Reply:
Ah, I just realized it's just the documentation that's wrong, not the code, in pyradiosky.